openstack_sdk/resources/compute.py

```diff
@@ -134,6 +134,46 @@ def get_server_password(self):
             ' password for this server: {0}'.format(server))
         return self.connection.compute.get_server_password(server)
 
+    def list_volume_attachments(self, query=None):
+        query = query or {}
+        self.logger.debug('Attempting to list servers')
```

Comment:
Attempting to list **volumes attachments**

Reply:
Done